qemu/tests/hotplug_virtio_mem.py

```diff
@@ -53,4 +54,13 @@ def run(test, params, env):
                                               threshold, vm, test)
         virtio_mem_utils.check_numa_plugged_mem(node_id, requested_size,
                                                 threshold, vm, test)
+    try:
+        hotplug_test.unplug_memory(vm, target_mem)
+    except QMPCmdError:
```

Comment:
@mcasquer  How about 
except QMPCmdError as e:
test.fail(XX) --> Pass this parameter "e" to this fail message as what you want.

Reply:
@menli820 thanks for the suggestion, but if you check in our internal test case, the step is expected to fail, that's the reason of the info message :grinning:

Reply:
Perhaps I can use the exception instance to look on it for the error message... what do you think?

Reply:
> Perhaps I can use the exception instance to look on it for the error message... what do you think?

@mcasquer  Thanks reminder, after checking our internal test case, it is reasonable with the test.log.info
  How about adding a **if**  judgment behind 'except QMPCmdError:' to confirm whether the exception message is expected?

Reply:
Code updated!